en/README.md

```diff
@@ -1128,6 +1138,17 @@ If fetch series of procedures' parameters are wrong or invalid and so on, it nee
 }
  ```
 
+### divideInfo
+
+In case of you use divideInfo with lastInfo, the lastInfo is saved if isLast is True. But if isLast is False, the lastInfo is destroyed.
```

Comment:
So,

"lastInfo is destroyed. " is wrong for specification.

"lastInfo is just ignored" seems to be appropriate here.